src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversion.cs

```diff
@@ -363,6 +364,17 @@ public bool IsMethodGroup
         }
 
         /// <summary>
+        /// Returns true if the conversion is an implicit tuple conversion.
+        /// </summary>
```

Comment:
I would add PROTOTYPE marker so we remember to link this to section in updated language spec (introducing new kind of conversion).